design/repo_maintenance_job_config.md

```diff
@@ -0,0 +1,144 @@
+# Repository maintenance job configuration design
+
+## Abstract
+Add this design to make the repository maintenance job can read configuration from a dedicate ConfigMap and make the Job's `PodSpec.Affinity` configurable. 
+
+## Background
+Repository maintenance is split from the Velero server to a k8s Job in v1.14 by [repository maintenance job](Implemented/repository-maintenance.md).
+The repository maintenance Job configuration was read from the Velero server CLI parameter.
+The Job inherits the most of the Job's PodSpec configuration from Velero server's Deployment's PodSpec.
+That inheritance also includes the Affinity setting.
+
+[node-agent affinity configuration](Implemented/node-agent-affinity.md) introduces affinity configuration for the node-agent DaemonSet pods.
+This design reuses the data structure introduced by that design to make the repository maintenance job can choose which node running on.
+
+## Goals
+- Let user can choose repository maintenance Job running on which nodes.
+- Unify the repository maintenance Job configuration at one place.
+
+## Non Goals
+- There was an [issue](https://github.com/vmware-tanzu/velero/issues/7911) to require the whole Job's PodSpec should be configurable. That's not in the scope of this design.
+- The introduced configuration is universal for all repositories. Configuration dedicated for each repository is not in this design's scope.
```

Comment:
The repo's configurations, either universal or dedicate, should always go into the backupRepository CR/configuration. So let's just add this clarification.
The current statement may deliver misleading message that the universal backup repo configurations may go into this repository maintenance job configuration instead of backupRepository CR/configuration.